Specs/DateTime/Spanish/DateTimePeriodExtractor.json

```diff
@@ -205,7 +205,8 @@
   },
   {
     "Input": "Regresaré por la noche",
-    "NotSupportedByDesign": "javascript",
+    "Comment": "In English only 'this night' is supported, 'at night' or 'by night' are not extracted as DateTimePeriod",
```

Comment:
Can you please create an issue to add support for "at night", "in the morning", etc. in EN and ES?

Reply:
done